### PR TITLE
V-1.0.0 of isizulu_ner_corpus

### PR DESCRIPTION
**Pull request by yvonnegitau**

### CODE REVIEW

File: ADD_NEW_DATASET.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This creates a copy of the code under your GitHub user account.
 
 Now you are ready, each time you want to add a new dataset, follow the steps in the following section:
```
Comment (lhoestq) [inline ADD_NEW_DATASET.md:24]: looks like your PR has changes in other files than `isizulu_ner_corpus`, could you please remove those ?

Also if it's a rebase/merge issue, feel free to create another branch and a new clean PR

File: datasets/msra_ner/msra_ner.py [hunk 2]
```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
```
Comment (lhoestq) [inline datasets/msra_ner/msra_ner.py:1]: Could you only include chanegs related to `isizulu_ner_corpus ` in this PR please ?

**Review by lhoestq — COMMENTED** [review]
Good job ! looks good to me :) 

It looks like you include changes from other files in this PR, can you fix that before we merge ?

File: ADD_NEW_DATASET.md (hunk 1 above)
Comment (yvonnegitau) [reply ADD_NEW_DATASET.md:24]: Seen the two languages have been added with the branch.  I assume no more actions is needed here

Comment (lhoestq) [reply ADD_NEW_DATASET.md:24]: Indeed ! thanks again for adding both of them.

Closing this PR then :)

**Comment by lhoestq** [comment]
closing since it's been added in #957

**Comment by lhoestq** [comment]
closing since it's been added in #957

**Pull request closed without merging**